main.ts

```diff
@@ -87,8 +87,15 @@ export default class ReadwiseMirror extends Plugin {
   headerTemplate: Template;
   highlightTemplate: Template;
 
-  private formatTags(tags: Tag[]) {
-    return tags.map((tag) => `#${tag.name}`).join(', ');
+  private formatTags(tags: Tag[], quote: boolean = false) {
+    // use unique list of tags

```

Comment:
Readwise might return tags with whitespace. These are not compatible with Markdown and Obsidian and therefore need to be sanitized. Replacing whitespace with a dash (-) seems a sensible choice that should not break frontmatter.